### DISCUSSION TITLE
Uncaught SyntaxError: missing : after property id

### DISCUSSION THREAD

**Discussion by kevinG73** [alpinejs/alpine#3154] [1. Help]
Hello ,
I get in the console : `Uncaught SyntaxError: missing : after property id` when i add x-data on my body 
![unknown](https://user-images.githubusercontent.com/23704479/189628461-0656ed9e-5d8d-4511-ae72-6d814cb6a64c.png)

<body x-data="{ sbm : 0 }"> using alpinejs 3 but it works with alpine 2
someone could explain ?

**Comment by SimoTod** [comment]
Can you post an example?
Usually, it's a syntax error in the x-data value: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Missing_colon_after_property_id

**Reply by kevinG73** [reply]
I use `Livewire` and `Laravel` , `blade ui` for displaying svg so i can't post on stackblitz or here here i wonder if there is a conflict with another library js
